content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
+As toolchains recomendadas nesta página **não requerem configuração para começar**.
 
-## You Might Not Need a Toolchain {#you-might-not-need-a-toolchain}
+## Você Pode Não Precisar de Uma Toolchain {#you-might-not-need-a-toolchain}
 
-If you don't experience the problems described above or don't feel comfortable using JavaScript tools yet, consider [adding React as a plain `<script>` tag on an HTML page](/docs/add-react-to-a-website.html), optionally [with JSX](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
+Se você não se depara com nenhum dos problemas descritos acima, ou se ainda não se sente confortável usando ferramentas JavaScript, considere [simplesmente adicionar uma tag `script` numa página HTML](/docs/add-react-to-a-website.html), opcionalmente [usando JSX](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
 
-This is also **the easiest way to integrate React into an existing website.** You can always add a larger toolchain if you find it helpful!
+Esta é **a maneira mais fácil de integrar React a um site já existente.** Você pode a qualquer momento adicionar à toolchain conforme achar útil.
 
-## Recommended Toolchains {#recommended-toolchains}
+## Toolchains Recomendadas {#recommended-toolchains}
 
-The React team primarily recommends these solutions:
+A equipe React recomenda principalmente as seguintes soluções:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
-- If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
+- Se você está **aprendendo React** ou **criando um novo [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app). 
+- Se você está fazendo um **site renderizado-no-servidor (SSR) com Node.js,** tente [Next.js](#nextjs).
+- Se você está fazendo um **site estático orientado a conteúdo,** experimente [Gatsby](#gatsby).
+- Se você está montando uma **biblioteca de componentes** ou **integrando com um código-base existente**, veja [Toolchains Mais Flexíveis](#more-flexible-toolchains).
 
 ### Create React App {#create-react-app}
 
-[Create React App](http://github.com/facebookincubator/create-react-app) is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
+[Create React App](http://github.com/facebookincubator/create-react-app) é um ambiente confortável para **aprender React**, e é a melhor maneira de começar uma **[single-page](/docs/glossary.html#single-page-application) application** em React. 
```

Comment:
let's stick with `um single-page` instead of `uma single-page` as you did above

Reply:
That's how I originally had it, but it was suggested otherwise. Oh well!